src/docs/community/rebrand.mdx

```diff
@@ -0,0 +1,27 @@
+import { Divider, FAQExpander } from '../../app/components';
+
+<!-- KEYWORDS: faq question answers -->
+
+# PX Blue is Rebranding
+
+In Q4 of 2021, we will be rebranding our design system from PX Blue to Brightlayer UI. This rebranding effort will impact all resources that PX Blue currently provides, including:
+
+-   NPM Packages
+-   Github Repositories
+-   Figma Stickersheet
+-   Documentation
+-   This documentation site & URL
```

Comment:
Can you distinguish between "Documentation" and "This documentation site & URL"? The way it is currently structured might cause some confusions...